gems/aws-sdk-core/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Feature - Add support for Observability.
```

Comment:
May want to mention the config option + otel support as part of the feature here.